src/hooks/useConfirmationText.ts

```diff
@@ -30,7 +33,7 @@ function useConfirmationText() {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [history.location.pathname]);
 
-  return confirmationText;
+  return { confirmationText, setConfirmationText };
```

Comment:
Isn't there a standard in React for returning an array? And then we could use one return value, or both, depending on our use case?

Reply:
Not really. `useState` returns an array, but hooks can return anything.
For example, `useTranslation()` for i18n returns an object and `useQuery` from Apollo does too.

I ultimately decided to return an object because we don't always need `confirmationText`. With an array we'd have an unused variable.
```
const [_confirmationText, setConfirmationText] = useConfirmationText();

// _confirmationText is unused sometimes

vs

const { setConfirmationText } = useConfirmationText();
```